src/components/MovieInfo/MovieInfo.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import './MovieInfo.css';
 
 const MovieInfo = (props) => {
+  console.log(props, 'movieinfo')
```

Comment:
We'll have to clear this on our next push, but real solid troubleshooting last night!